WordPress/Classes/Models/Post.m

```diff
@@ -141,4 +167,115 @@ - (BOOL)hasLocalChanges
     return NO;
 }
 
+#pragma mark - WPPostContentViewProvider Methods
+
+- (NSString *)authorNameForDisplay
+{
+    return self.author;
+}
+
+- (NSURL *)blogURL
+{
+    return [NSURL URLWithString:self.blog.url];
+}
+
+- (NSString *)blogURLForDisplay
+{
+    return self.blog.displayURL;
+}
+
+- (NSInteger)commentCount
+{
+    return 0;
```

Comment:
Annoying note: Maybe a comment that outlines why this getter and the ones below it return a constant?